vm-allocator/src/system.rs

```diff
@@ -68,8 +66,6 @@ impl SystemAllocator {
         #[cfg(target_arch = "x86_64")] io_size: GuestUsize,
         platform_mmio_base: GuestAddress,
         platform_mmio_size: GuestUsize,
-        mmio_hole_base: GuestAddress,
-        mmio_hole_size: GuestUsize,
```

Comment:
Nit: you forgot to remove the comments about the deleted inputs.

Reply:
Updated.